README.md

```diff
@@ -20,21 +17,21 @@ git clone https://github.com/MLukman/Keycloak-PII-Data-Encryption-Provider.git K
 cd Keycloak-PII-Data-Encryption-Provider
 ```
 
-Compile this provider into a JAR file using the following command. JDK 17 or above and Maven are required to be pre-installed on the machine.
+Compile this provider into a JAR file using the following command. JDK 17 or above and Maven are required to be pre-installed on the machine. Make sure to match the `keycloak.version` parameter with the version of the target Keycloak.
 
 ```shell 
-mvn clean package
+mvn clean package -Dkeycloak.version=26.1.1
 ```
 
-Copy paste the packaged JAR file from inside `target` folder into Keycloak's `providers` folder. Run `kc.sh build` command to get Keycloak to register this provider.
+Copy paste the packaged JAR file from inside `target` folder into Keycloak `providers` folder. Run `kc.sh build` command to get Keycloak to register this provider.
 
 ### Install inside Docker Image
 
 Use this method if this provider needs to be pre-packaged inside a custom Keycloak Docker image. Below is a sample Dockerfile:
 
 ```dockerfile
 # ARG defined before FROM in multi-staged Dockerfile is shared among the stages
-ARG KEYCLOAK_VERSION=26.0.6
+ARG KEYCLOAK_VERSION=26.1.1
```

Comment:
I think it is the wrong version.  Maybe use a place holder to not have to update each time.

Reply:
Yes, this comes from the original repository and it's onl in the readme, that's why I left it untouched..